dev/tidb-in-kubernetes/deploy/aws-eks.md

```diff
@@ -254,7 +252,35 @@ Terraform 脚本中为运行在 EKS 上的 TiDB 集群提供了合理的默认
 
 作为例子，默认集群使用了 `./default-cluster.yaml` 作为 `values.yaml` 配置文件，并在配置中打开了"配置文件滚动更新"特性。
 
-值得注意的是，在 EKS 上部分配置项无法在 `values.yaml` 中进行修改，包括集群版本、副本数、`NodeSelector` 以及 `Tolerations`。这些配置项由 Terraform 直接管理以确保基础设施与 TiDB 集群之间的一致性。你可以通过修改 `cluster.tf` 文件中的 `tidb-cluster` module 参数来自定义这些配置。
+值得注意的是，在 EKS 上部分配置项无法在 `values.yaml` 中进行修改，包括集群版本、副本数、`NodeSelector` 以及 `Tolerations`。`NodeSelector` 和 `Tolerations` 由 Terraform 直接管理以确保基础设施与 TiDB 集群之间的一致性。集群版本和副本数可以通过 `cluster.tf` 文件中的 `tidb-cluster` module 参数来修改。
+
+> **注意：**
+>
+> 自定义 `values.yaml` 配置文件中，不能包含如下配置（`tidb-cluster` module 固定配置）：
```

Comment:
These configurations are recommended but customizable

Reply:
Actually, it's not easy to customize them, for example, if customize the storage class name, users have to create it first, anyway, I will change the description as "It's not recommended to include following configurations in customized values.yaml".